stores/institutions.js

```diff
@@ -304,5 +304,11 @@ export const useInstitutionStore = defineStore("institution", {
         console.log("NO TEAMS", e);
       }
     },
+    async deleteInstitution(id) {
```

Comment:
```suggestion
    deleteInstitution(id) {
```
This doesn't use `await` so no need to make it async.